lib/posix-user/user.c

```diff
@@ -110,7 +110,7 @@ int seteuid(uid_t euid __unused)
 	return 0;
 }
 
-int getresuid(uid_t *ruid __unused, uid_t *euid __unused, uid_t *suid __unused)
+UK_SYSCALL_R_DEFINE(int, getresuid, uid_t*, ruid, uid_t*, euid, uid_t*, suid)
 {
 	return 0;
```

Comment:
Not really related to the purpose of this PR, but shouldn't there be something like `*ruid = *euid = *suid = UK_DEFAULT_UID;` before returning?